Makefile

```diff
@@ -38,7 +39,7 @@ export PATH := $(PATH):$(CURDIR)
 endif
 
 # The only thing we really want to do is build
-all:	check_mdbook check_mdbook_linkcheck check_mdbook_pagetoc check_prettier
+all:	check_mdbook check_mdbook_linkcheck check_mdbook_linkcheck check_mdbook_pagetoc check_prettier
```

Comment:
Was it intentional to list `check_mdbook_linkcheck` twice? Seems like a mistake?